.github/workflows/release.yml

```diff
@@ -52,13 +41,19 @@ jobs:
           bun install --frozen-lockfile
           bun add -g npm
 
-      - name: 🔄 Update version
+      - name: 🔧 Configure Git
         run: |
           git config user.name 'github-actions[bot]'
           git config user.email 'github-actions[bot]@users.noreply.github.com'
-
           git config --global push.followTags true
-          npm version ${{ env.VERSION_CHANGE }} -m "Release %s"
+
+      - name:  Create GitHub release notes
+        run: |
+          bunx changelogen github release --token ${{ secrets.GITHUB_TOKEN }}
+
+      - name: 📜 Generate changelog and release notes
+        run: |
+          bunx changelogen --release --push
```

Comment:
Can it all be compressed into one step?

Reply:
No, these are two separate steps. I've checked [their code](https://github.com/unjs/changelogen/blob/main/src/cli.ts#L5) and also other people's repositories which are using it, e.g. https://github.com/schoero/eslint-plugin-readable-tailwind/blob/main/package.json#L50. And they are calling it separately.

Reply:
I can combine both commands into a single step.

Reply:
First I thought, that you mean the command itself

Reply:
Do you know if the first command fails, will the second be executed? I don't want to execute it, if the GH release wasn't working. So with two separate steps it would be working, but with two separate commands in a single step, I'm not pretty sure at the moment

Reply:
According to Copilot this should work:

No, the second command will not be executed if the first command fails. In a GitHub Actions workflow, commands in a step are executed sequentially, and if one command fails, the subsequent commands in that step will not be executed.

I've updated it.